src/Web3ProviderBackend.ts

```diff
@@ -388,3 +406,34 @@ function without<T>(list: T[], item: T): T[] {
   }
   return list
 }
+
+// Allowed keys for a JSON-RPC transaction as defined in:
+// https://ethereum.github.io/execution-apis/api-documentation/
+const allowedTransactionKeys =  ['accessList', 'chainId', 'data', 'from', 'gas', 'gasPrice', 'maxFeePerGas',
+  'maxPriorityFeePerGas', 'nonce', 'to', 'type', 'value']
+
+// Convert a JSON-RPC transaction to an ethers.js transaction.
+// The reverse of this function can be found in the ethers.js library:
+// https://github.com/ethers-io/ethers.js/blob/v5.7.2/packages/providers/src.ts/json-rpc-provider.ts#L701
+function convertTransaction(tx: { [key: string]: any }): ethers.providers.TransactionRequest {
```

Comment:
Even though return type is specified, it is not very clear what the input and the output are. Maybe something like `convertTxObjectToEthersTxRequest`?